docs/configuring-openshift.md

```diff
@@ -0,0 +1,79 @@
+# Configuring OpenShift
+The following sections describe how Minishift can be used to configure OpenShift. For details about managing the Minishift VM refer to the [Using Minishift](./using.md) section.
+
+<!-- MarkdownTOC -->
+
+- [Updating OpenShift configuration](#updating-openshift-configuration)
+- [Persistent volumes](#persistent-volumes)
+- [Private container registries](#private-container-registries)
+
```

Comment:
When I checked the rendered pages I saw an extra empty line between `Persistent volumes` and `Private container registries` , not sure why. You might want to take a look.